core/src/main/java/io/grpc/internal/GrpcAttributes.java

```diff
@@ -37,6 +38,10 @@
   public static final Attributes.Key<Map<String, ?>> NAME_RESOLVER_SERVICE_CONFIG =
       Attributes.Key.create("service-config");
 
+  @NameResolver.ResolutionResultAttr
+  public static final Attributes.Key<List<EquivalentAddressGroup>> ATTR_LB_ADDRS =
```

Comment:
it makes sense it is here. but it is quite different than [proposal](https://github.com/grpc/proposal/blob/master/A26-grpclb-selection.md). can you update the wrong one?

Reply:
Now we will have two `ATTR_LB_ADDRS` keys, one in `grpclb` and one in `core`. They point to the same key. `DnsNameResolver` in `core` sets this attribute and `GrpclbLoadBalancer` will access it. In the next change, logics for querying SRV will be moved to `grpclb`, so only `grpclb`'s resolver will set `ATTR_LB_ADDRS` attribute. Then we can eliminate the key in `core`.

Reply:
Done.